02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Ben's Site</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div>
+<img src="https://avatars2.githubusercontent.com/u/6634962?v=3&s=460"/>
+  </div>
+
+  <div>
+<h1 style="font-family: cursive">Benjamin Blowe</h1>
+  </div>
+<p>Born here in San Antonio, spent my childhood living in Florida, and have been back in SATX for the last decade. I'm an anthropology student at UTSA, I currently work in customer service, and I'm here at ACA to make a career in web development.</p>
+
+  <div>
+<h4>Contact:</h4>
+<h4>bdbblowe@gmail.com | (210) 487-8118</h4>
+</div>
+
+  <div>
+<img src="http://docs.whitesourcesoftware.com/download/attachments/17989744/github_logo.png?version=1&modificationDate=1463491374000" alt="Github">
```

Comment:
Was this supposed to be a hyperlink? I think it would be a good idea as it would take potential employers to your GitHub page and see all the cool projects you have completed. Have a look at this demo on jsfiddle https://jsfiddle.net/robertonavarro/xmp8r6sp/